plugins/ouch/hm-module.nix

```diff
@@ -0,0 +1,49 @@
+{
+  options =
+    { mkKeyOption, ... }:
+    _: {
+      keys.compress = mkKeyOption {
+        on = [ "C" ];
+        run = "plugin ouch";
+        desc = "Compress with ouch";
+      };
+    };
+  config =
+    { cfg, setKeys, ... }:
+    { lib, pkgs, ... }:
+    lib.mkMerge [
+      (setKeys cfg.keys)
+      {
+        programs.yazi = {
+          yaziPlugins.runtimeDeps = [ pkgs.ouch ];
+          settings.plugin = {
+            prepend_previewers =
+              builtins.map
+                (format: {
+                  mime = "application/${format}";
+                  run = "ouch";
+                })
+                [
```

Comment:
can you maybe similarly to this generate options to disable individual previewers?